frontstage/templates/surveys/surveys-access.html

```diff
@@ -30,7 +30,8 @@ <h2 class="neptune">Download</h2>
 
 
 <a id="download-survey-button" class="btn action-button btn--primary" target="_blank"
-   href="{{ url_for('surveys_bp.download_survey', cid=collection_instrument_id, case_id=case_id) }}">
+   href="{{ url_for('surveys_bp.download_survey', cid=collection_instrument_id, case_id=case_id) }}"
```

Comment:
Doesn't look like cid=collection_instrument_id is actually needed. download_survey just requires case_id.